### PR TITLE
Alter column name for receipt_cids table.

### PR DESCRIPTION
**Pull request by arijitAD**
https://github.com/vulcanize/go-ethereum/pull/119
https://github.com/vulcanize/ipld-eth-server/pull/103

### CODE REVIEW

File: db/migrations/00027_rename_cid_and_mh_key_in_eth_receipt_cids.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
+-- +goose Up
+ALTER TABLE eth.receipt_cids
+RENAME COLUMN cid TO leaf_cid;
+
+ALTER TABLE eth.receipt_cids
+RENAME COLUMN mh_key TO leaf_mh_key;
+
+ALTER INDEX eth.rct_mh_index RENAME TO rct_leaf_cid_index;
+ALTER INDEX eth.rct_cid_index RENAME TO rct_leaf_mh_key_index;
```
Comment (i-norden) [inline db/migrations/00027_rename_cid_and_mh_key_in_eth_receipt_cids.sql]: Let's put the RENAMEs on a new line like with the tables

File: db/schema.sql [hunk 2]
```diff
@@ -104,6 +102,33 @@ end;
 $_$;
 
 
+--
+-- Name: canonical_header(bigint); Type: FUNCTION; Schema: public; Owner: -
+--
+
+CREATE FUNCTION public.canonical_header(height bigint) RETURNS integer
```
Comment (i-norden) [inline db/schema.sql]: We should add "down" migrations to remove this at some point

File: db/schema.sql [hunk 3]
```diff
@@ -218,11 +254,33 @@ BEGIN
...
+  RETURN new_child_result;
 END
 $$;
 
 
+--
+-- Name: header_weight(character varying); Type: FUNCTION; Schema: public; Owner: -
+--
+
+CREATE FUNCTION public.header_weight(hash character varying) RETURNS bigint
```
Comment (i-norden) [inline db/schema.sql]: Ditto, we don't use this anymore

**Review by i-norden — APPROVED** [review]
LGTM

File: db/migrations/00027_rename_cid_and_mh_key_in_eth_receipt_cids.sql (hunk 1 above)
Comment (arijitAD) [reply db/migrations/00027_rename_cid_and_mh_key_in_eth_receipt_cids.sql]: Done

File: db/schema.sql (hunk 2 above)
Comment (arijitAD) [reply db/schema.sql]: Done

File: db/schema.sql (hunk 3 above)
Comment (arijitAD) [reply db/schema.sql]: Done

**Pull request merged**